recipes/nintendo/3ds

```diff
@@ -62,7 +62,7 @@ stella libretro-stella https://github.com/stella-emu/stella.git master NO GENERI
 test libretro-samples https://github.com/libretro/libretro-samples.git master YES GENERIC Makefile tests/test
 theodore libretro-theodore https://github.com/Zlika/theodore.git master YES GENERIC Makefile .
 tgbdual libretro-tgbdual https://github.com/libretro/tgbdual-libretro.git master NO GENERIC Makefile .
-tic80 libretro-tic80 https://github.com/nesbox/TIC-80.git master YES CMAKE Makefile builddir -DBUILD_PLAYER=OFF -DBUILD_SOKOL=OFF -DBUILD_SDL=OFF -DBUILD_DEMO_CARTS=OFF -DBUILD_LIBRETRO=ON -DCMAKE_C_COMPILER=${DEVKITARM}/bin/arm-none-eabi-gcc -DCMAKE_CXX_COMPILER=${DEVKITARM}/bin/arm-none-eabi-g++ -DCMAKE_SYSTEM_PROCESSOR=arm -DCMAKE_SYSTEM_NAME=Linux -DCMAKE_CROSSCOMPILING=TRUE -DCMAKE_SHARED_LIBRARY_LINK_C_FLAGS=-fPIC -DCMAKE_C_LINK_FLAGS=-fPIC -DCMAKE_TRY_COMPILE_TARGET_TYPE=STATIC_LIBRARY -DDISABLE_NETWORKING=TRUE -DLIBRETRO_STATIC=TRUE -DLIBRETRO_SUFFIX=_ctr -DCMAKE_CXX_FLAGS=" -march=armv6k -mtune=mpcore -mfloat-abi=hard" -DCMAKE_C_FLAGS=" -march=armv6k -mtune=mpcore -mfloat-abi=hard"
+tic80 libretro-tic80 https://github.com/nesbox/TIC-80.git master YES CMAKE Makefile builddir -DBUILD_PLAYER=OFF -DBUILD_SOKOL=OFF -DBUILD_SDL=OFF -DBUILD_DEMO_CARTS=OFF -DBUILD_LIBRETRO=ON -DDISABLE_NETWORKING=TRUE -DLIBRETRO_STATIC=TRUE -DLIBRETRO_SUFFIX=_ctr
```

Comment:
`LIBRETRO_STATIC`... Should we just have that always default to `TRUE` in the TIC-80 cmake definition?